traefik-mixin/alerts/alerts.libsonnet

```diff
@@ -0,0 +1,33 @@
+{
+  groups+: [
+    {
+      name: 'traefik',
+      rules: [
+        // TraefikConfigReloadFailuresIncreasing
+        {
+          alert: 'TraefikConfigReloadFailuresIncreasing',
+          expr: "sum(rate(traefik_config_reloads_failure_total[5m])) > 0",
```

Comment:
sum would drop any labels, making it hard to identify which speific traefik instance has failed to reload. Would it work without using sum()?

If grouping is neceassrry, I suggest to add variable `groupLabels` in config.libsonnet file and use this variable for aggregation i.e. `sum by (%(groupLabels)s)` where groupLabels could be labels like job, cluster, etc dependending on traefik deployment

Reply:
Yup good idea for customizable group labels. Perhaps I can think of a sum-less version too

Reply:
I stuck with sum but adding labels for both sets of alerts that are configurable 932c72cce9949522402b78e7813af36aa8d62ea3